model/trigger/task.go

```diff
@@ -297,15 +297,15 @@ func (t *taskTriggers) taskRegression(sub *event.Subscription) (*notification.No
 }
 
 func isTaskRegression(t *task.Task) (bool, *alertrecord.AlertRecord, error) {
-	shouldNotify := false
+	isRegression := false
```

Comment:
It seems like every time you use this, you return false except for the last line of the program, where you return true. To an unfamiliar reader, it's harder to follow because it makes you think that something complicated is going on. Can you just get rid of this variable?

Reply:
done